### DISCUSSION TITLE
Automation of labels

### DISCUSSION THREAD

**Discussion by HimangshuSharma01** [sanjay-kv/Open-source-Practice#1153] [Ideas]
@sanjay-kv Sir we can add labels automatically to level 1 and later on change it if required

**Comment by sanjay-kv** [comment]
yes we can do that. using workflow. here by default we can add level `leve1` @HimangshuSharma01

**Reply by HimangshuSharma01** [reply]
> yes we can do that. using workflow. here by default we can add level `leve2` @HimangshuSharma01 

Okay sir  In the project of git hub awesome profile?  @sanjay-kv

**Reply by sanjay-kv** [reply]
yes

**Comment by HimangshuSharma01** [comment]
![image](https://github.com/user-attachments/assets/fe12f0f5-4cb8-4a37-9ee2-8943efebd4ac)
Sir created the auto label

**Reply by HimangshuSharma01** [reply]
![Screenshot 2024-10-17 120014](https://github.com/user-attachments/assets/b61fe35f-01fa-46ee-bbde-5f97ded5e233)
![Screenshot 2024-10-17 120008](https://github.com/user-attachments/assets/d6bc2911-295b-41e3-94bb-a10d30a3300b)
@sanjay-kv  It is working

**Reply by sanjay-kv** [reply]
@HimangshuSharma01 awesome, i saw this. great work. add labels for your PR as well. so it will count to gssoc leaderboard

**Reply by HimangshuSharma01** [reply]
> @HimangshuSharma01 awesome, i saw this. great work. add labels for your PR as well. so it will count to gssoc leaderboard

Yes sir

**Comment by HimangshuSharma01** [comment]
@sanjay-kv sir can i assign them?
![Screenshot 2024-10-17 194005](https://github.com/user-attachments/assets/99d84e57-1927-4cbe-9408-e90a3be982af)

**Reply by sanjay-kv** [reply]
yes

**Comment by HimangshuSharma01** [comment]
@sanjay-kv sir there is some kind of double badge
![Screenshot 2024-10-17 215256](https://github.com/user-attachments/assets/9e859bce-3b0c-450d-ba76-11c2ae6035e1)
![Screenshot 2024-10-17 215246](https://github.com/user-attachments/assets/7a00a7a4-badc-44d5-89b9-0d46daa59b90)

**Reply by HimangshuSharma01** [reply]
can i remove the levels with no points?

**Reply by sanjay-kv** [reply]
yes please
